superset/views/core.py

```diff
@@ -1919,7 +1945,7 @@ def dashboard(self, dashboard_id):
         else:
             qry = qry.filter_by(slug=dashboard_id)
 
-        dash = qry.one()
+        dash = qry.first()#one()
```

Comment:
Importing a new dashboard from an existing dashboard leaves the slug value the same and when .one has more than one result, it errors out. 
Would it be better to avoid importing dashboards with the same slug? or to add a random new slug or just return the first slug?